scripts/apps/ingest/index.js

```diff
@@ -201,4 +201,12 @@ angular.module('superdesk.apps.ingest', [
             label: 'Wufoo Forms',
             templateUrl: asset.templateUrl('apps/ingest/views/settings/wufoo.html')
         });
+        ingestSources.registerFeedingService('ritzau', {
+            label: 'Ritzau Webservice',
+            templateUrl: asset.templateUrl('apps/ingest/views/settings/ritzauConfig.html')
+        });
+        ingestSources.registerFeedingService('ntb_ritzau', {
```

Comment:
is there anything ntb specific on the service?

Reply:
just small stuff, the same that we do for other feeds (default category, utenriks or sport), but still it is a specific behaviour.

Reply:
isn't it done with the parser?

Reply:
Oh right, got confused between service and parser :-/ . Will fix that thanks!